test/app_test_case.py

```diff
@@ -58,7 +63,24 @@ def address(self):
         return self.server_wrapper.server.server_address
 
 
-class LocalAppTestCase(CatalogTestCase, metaclass=ABCMeta):
+class LambdaEnvTestCase(AzulUnitTestCase):
+    _lambda_env_mock = None
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls._lambda_env_mock = patch.dict(os.environ,
+                                          AWS_LAMBDA_FUNCTION_NAME='unit-tests',
+                                          azul_aws_account_name='test-hca-dev')
```

Comment:
There is already a test case base class that deals with mocking account info. Maybe this class should be dissolved into that one.

Reply:
It looks like the account info mocking is done in `AzulUnitTestCase`. I don't think `LambdaEnvTest` should be dissolved into `AzulUnitTestCase` because not every unit test covers code that is run in a Lambda. It might make more sense to dissolve `LambdaEnvTestCase` downwards into `LocalAppTestCase` instead, but then `TestLogForwarding` would need to be re-written to inherit from `LocalAppTestCase`.

Reply:
Please request PL slot about this.

Reply:
Decided at PL to try merging `LambdaEnvTestCase` either up (into `AzulUnitTestCase`) or down (into `LocalAppTestCase`) depending on which is easier

Reply:
Moving the AWS account mocks from `AzulUnitTestCase` to `LocalAppTestCase` caused failures in 

- service.test_portal_service.TestPortalService
- service.test_storage_service.StorageServiceTest
- test_deployment.TestDeploymentAWS

Reply:
Moving the Lambda environment mock up into `AzulUnitTestCase` also breaks `test_deployment.TestDeploymentAWS`. This is easy to fix so I'm going with this option.